docs/server/configuration/plugins.md

```diff
@@ -5,7 +5,9 @@ order: 6
 
 # Plugins configuration
 
-The commercial edition of EventStoreDB ships with several plugins that augment the behavior of the open source server. Each plugin is documented in relevant sections. For example, under Security, you will find the User Certificates plugin documentation.
+EventStoreDB ships with several plugins that augment the behavior of the open source server. Each plugin is documented in relevant sections. For example, under Security, you will find the User Certificates plugin documentation.
```

Comment:
> open source server
We are not open source anymore: we are source available
Open Source can only be used when  using an OSI approved licence 
I'm unsure what the correct term is , @alexeyzimarev  any idea ?

Reply:
We discussed it, will be fixed.